packages/kit/test/apps/basics/src/routes/routing/_tests.js

```diff
@@ -188,15 +188,64 @@ export default function (test, is_dev) {
 	test(
 		'back button returns to previous route when previous route has been navigated to via hash anchor',
 		'/routing/hashes/a',
-		async ({ page, clicknav, back }) => {
-			await clicknav('[href="#hash-target"]');
+		async ({ page, clicknav }) => {
+			await page.click('[href="#hash-target"]');
 			await clicknav('[href="/routing/hashes/b"]');
 
-			await back();
+			await page.goBack();
```

Comment:
Changed this to `page.click` and `page.goBack` since the `clicknav` and `back` utils rely on `sveltekit:navigation-end` to resolve, which these hash-only anchor tags would skip, resulting in timeout.

Using playwright's builtin functions to avoid it.